docker/Dockerfile

```
@@ -16,7 +16,8 @@
 # Keep these argument defaults in sync with GitHub vars.RUST_PROD_FEATURES and vars.RUST_TEST_FEATURES
 # https://github.com/ZcashFoundation/zebra/settings/variables/actions
 ARG FEATURES="default-release-binaries"
-ARG TEST_FEATURES="lightwalletd-grpc-tests zebra-checkpoints"
+ARG TEST_FEATURES="lightwalletd-grpc-tests zebra-checkpoints shielded-scan"
```

I think we are better off not including `shielded-scan` for now, actually.

We will use it at some point, for example, when we make it a part of the default features, but we now use it only for a single test and enable it for the test separately.

```suggestion
ARG TEST_FEATURES="lightwalletd-grpc-tests zebra-checkpoints"
```

---

Solved with [fix(docker): do not add shielded-scan as part of TEST_FEATURES](https://github.com/ZcashFoundation/zebra/pull/8175/commits/c9f5118ee3b7edf5bdf39e9c35d79679d89841a0)